web/app/js/components/Tap.jsx

```diff
@@ -85,7 +86,7 @@ class Tap extends React.Component {
   onWebsocketClose = e => {
     this.stopTapStreaming();
 
-    if (!e.wasClean) {
+    if (!e.wasClean && e.code !== 1006) {
```

Comment:
can we leave a comment here explaining why we're ignoring this? (maybe a link to the ticket would be helpful)